nmostesting/GenericTest.py

```diff
@@ -20,20 +20,28 @@
 import inspect
 import uuid
 import time
-from authlib.jose import jwt
+import ssl
+import threading
 
 from . import TestHelper
 from .NMOSUtils import NMOSUtils
 from .Specification import Specification
 from .TestResult import Test
 from . import Config as CONFIG
+from flask import Flask
+from .mocks.Auth import AUTH_API, PRIMARY_AUTH, SECONDARY_AUTH
+from OpenSSL import crypto
+from cryptography.hazmat.primitives import serialization
+from cryptography import x509
+from werkzeug.serving import make_server
```

Comment:
I feel this hints that `GenericTest` is being polluted with the mechanics of the mock Auth server - I would rather see this functionality factored into `Auth.py` or a supporting `IS10Utils.py` module.

Reply:
Done, in `Auth.py`